jdat_notebooks/README.md

```diff
@@ -1,237 +1,13 @@
-once the notebook is public (both in and out of STScI).  While the repo maintainers manage this process, the notebook
-authors may be called in from time to time to provide opinions or perspectives on any proposed changes.
-
-# Procedure to submit a notebook as a Pull Request
-
-Submission of a new notebook follows the Github Pull Request workflow.  All details are in the
-[STScI git workflow style guide](https://github.com/spacetelescope/style-guides/blob/master/guides/git-workflow.md).
-Here we give a "cookbook" procedure,
-but do not hesitate to reach out for help from other members of the team if you are stuck or are not sure how
-it is supposed to work!
-
-Note also that these steps are slightly different for if you update a notebook after you've created the first pull request - you can skip steps 1-3 and 5.
-
-1. Go to the github working space https://github.com/spacetelescope/dat_pyinthesky and fork the repository to your user account
-(button "Fork" in the top right corner).
-
-2. Clone the repository locally on your machine
-
-``git clone git@github.com:username/dat_pyinthesky.git``
-
-3. While this sets up ``origin`` to point to your fork, there is currently no connection to the main ``spacetelescope`` "upstream" repository.  So you can point your local clone to the right repository by doing:
-
-``git remote add upstream https://github.com/spacetelescope/dat_pyinthesky.git``
-
-4. Create a new branch where to start the development and move to that branch
-
-``git branch new_notebook_branch_name``
-
-``git checkout new_notebook_branch_name``
-
-5. Create a new folder where to develop the notebook
-
-``cd jdat_notebook``
-
-``mkdir new_notebook_name``
-
-6. Now start building your notebook (new_notebook_name.ipynb)!
-
-7. At any point in the development, save your work and push it up to your forked repository. (Important: you must clear the outputs on your notebook using the Jupyter interface before doing an ``add/commit`` like this.)
-
-``git add new_notebook_name.ipynb``
-
-``git commit -m "Clear message to state the fix or improvement to the notebook"``
-
-``git push origin new_notebook_branch_name``
-
-(sometimes you have to reset the upstream, so in that case it is ``git push --set-upstream origin new_notebook_branch_name``)
-
-8. When you are happy with your notebook, double check that you have satisfied the thecnical requirements of the specific status
-of your notebook (see above).
-
-9. Now you can create a Pull Request from the ``spacetelescope/dat_pyinthesky`` repository. You do that
-by clicking on ``New pull request`` on the webpage, then the link ``compare across forks``. Then set the base repository
-to ``spacetelescope/dat_pyinthesky`` and branch ``master`` and the head fork to
-the branch on your personal fork, so repository ``username/dat_pyinthesky`` and branch ``new_notebook_branch_name``. You
-set a title and you click on ``Create pull request``.
-
-One of the team members can then merge your Pull Request.
```

Comment:
Did you really mean to nuke most of the text in the Readme.md file, which gives instructions to notebook leads?

Reply:
Should this at least link to contributing.md?

Reply:
Yes, I meant to nuke it and move it to contributing.md.  And yes, see Erik's comments below.